doc/rados/operations/stretch-mode.rst

```diff
@@ -54,32 +53,32 @@ guarantees.
 The second important category of failures is when you think you have data replicated
 across data centers, but the constraints aren't sufficient to guarantee this.
 For instance, you might have data centers A and B, and your CRUSH rule targets 3 copies
-and places a copy in each data center with a min_size of 2. The PG may go active with
+and places a copy in each data center with a ``min_size`` of 2. The PG may go active with
 2 copies in site A and no copies in site B, which means that if you then lose site A you
 have lost data and Ceph can't operate on it. This situation is surprisingly difficult
 to avoid with standard CRUSH rules.
 
 Stretch Mode
 ============
-The new stretch mode is designed to handle the 2-site case. (3 sites are
-just as susceptible to netsplit issues, but much more resilient to surprising
-data availability ones than 2-site clusters are.)
+The new stretch mode is designed to handle the 2-site case. Three sites are
+just as susceptible to netsplit issues, but are much more tolerant of
+component availability outages than 2-site clusters are.
 
 To enter stretch mode, you must set the location of each monitor, matching
-your CRUSH map. For instance, to place mon.a in your first data center ::
+your CRUSH map. For instance, to place ``mon.a`` in your first data center ::
 
   $ ceph mon set_location a datacenter=site1
 
 Next, generate a CRUSH rule which will place 2 copies in each data center. This
-will require editing the crush map directly::
+will require editing the CRUSH map directly::
 
   $ ceph osd getcrushmap > crush.map.bin
   $ crushtool -d crush.map.bin -o crush.map.txt
 
-Then edit the crush.map.txt file to add a new rule. Here
+Now edit the ``crush.map.txt`` file to add a new rule. Here
 there is only one other rule, so this is id 1, but you may need
-to use a different rule id. We also have two data center buckets
-named site1 and site2::
+to use a different rule id. We also have two datacenter buckets
```

Comment:
```suggestion
to use a different rule id. We also have two data center buckets
```

Reply:
Ah, here's an example of what prompted my desire to harmonize the other way.  This line refers to a defined CRUSH bucket type, whose name decisively does not contain a space, so we have to refer to it that way.  Ie, this is a proper noun in a way.  Other usages of `data*center` refer to a building/hall/room that is used in a certain way.  

https://github.com/ceph/ceph/blob/master/doc/rados/operations/crush-map.rst#types-and-buckets

This is part of why I favored harmonizing on `datacenter`, to be consistent with the CRUSH bucket type.

`id` though I need to uppercase, as we're talking about CRUSH not Walter Pidgeon.

Reply:
In this case, I think you're right. The IBM Style Guide wasn't written with CRUSH bucket types in mind, and your impulse to refer to them correctly is correct.

Crazy reference to Walter Pidgeon being injured, by the way.

Reply:
_Forbidden Planet_, "monsters from the id" 8^)